src/main/java/frc/robot/commands/auto/GridNineTwoPiece.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands.auto;
+
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.commands.scoring.PositionIntake;
+import frc.robot.commands.scoring.PositionIntake.Position;
+import frc.robot.commands.scoring.elevator.MoveElevator;
+import frc.robot.commands.scoring.intake.IntakeGamePiece;
+import frc.robot.constants.ElevatorConstants;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.Elevator;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Wrist;
+
+public class GridNineTwoPiece extends SequentialCommandGroup {
+  public GridNineTwoPiece(Drivetrain drive, Elevator elevator, Wrist wrist, Intake intake) {
+    addCommands(
+      new AutoDeposit(Position.TOP, false, elevator, wrist, intake),
```

Comment:
is the reason why the intake dosen't stow in this command is to save time?

Reply:
yes